sixtyfps_runtime/interpreter/api.rs

```diff
@@ -21,6 +21,7 @@ pub use sixtyfps_corelib::window::api::Window;
 /// the contained values.
 #[derive(Debug, Copy, Clone, PartialEq)]
 #[repr(i8)]
+#[non_exhaustive]
```

Comment:
maybe the `Other` should be `#[doc(hidden)]` and handle with the `_` pattern anyway?

Reply:
👍